content/documentation/runtimes-monitoring/index.adoc

```diff
@@ -103,6 +103,17 @@ As an example and for self-monitoring purpose Kiali itself link:https://github.c
 
 The pod annotation for Kiali is: `kiali.io/runtimes: go`
 
+
+=== Envoy
+
+The Envoy sidecars that come with Istio are also supplying link:https://www.envoyproxy.io/docs/envoy/latest/configuration/upstream/cluster_manager/cluster_stats[some internal metrics], that you can see in Kiali. They are not the same than the metrics reported by Istio Telemetry, and of which Kiali makes an intensive usage. Some of them can be redundant.
```

Comment:
This is not bad, here are two options:
Required change: 
`"not the same than" -> "not the same as"`
Optional change:
`Istio's Envoy sidecars supply <link>[some internal metrics] that can be viewed in in Kiali.  They are different than the metrics reported by Istio Telemetry, of which Kiali uses intensively.  Some of Envoy's metrics may be redundant.`

Reply:
They are different than the metrics reported by Istio Telemetry, of which Kiali uses intensively.
I would revise as
They are different than the metrics reported by Istio Telemetry, which Kiali uses extensively.